src/pages/index.js

```diff
@@ -23,26 +25,27 @@ const config = {
 
```

Comment:
From the 8th sprint all constants (`initialCards` and `config`) should be in file `constants.js` in folder `utils` to clean up the main file.